--- A:ПриСозданииНаСервере
+++ F:ПриСозданииНаСервере
@@ -25,8 +25,8 @@
 		КонецЕсли;
 	КонецЕсли;
 	
-	АдресПродВИМ = "!http://amapfront/PROD_AVC_FO-DL/ws/avancore_ws.1cws?wsdl";
-	АдресПродБФ = "!http://AMAPFRONT/PROD_BFAM_FO-DL/ws/avancore_ws.1cws?wsdl";
+	АдресПродВИМ = "http://amapfront/PROD_AVC_FO-DL/ws/avancore_ws.1cws?wsdl";
+	АдресПродБФ = "http://AMAPFRONT/PROD_BFAM_FO-DL/ws/avancore_ws.1cws?wsdl";
 	
 	Попытка
 		ПараметрыПодключения = ОбщегоНазначенияКлиентСервер.ПолучитьПараметрыПодключенияИзСтрокиСоединенияИнформационнойБазы(СтрокаСоединенияИнформационнойБазы());