--- ALEKSEY/SettingsForm_Module.bsl
+++ FEDOR/SettingsForm_Module.bsl
@@ -47,6 +47,6 @@
 	КонецЕсли;
 	
-	АдресПродВИМ = "!http://amapfront/PROD_AVC_FO-DL/ws/avancore_ws.1cws?wsdl";
-	АдресПродБФ = "!http://AMAPFRONT/PROD_BFAM_FO-DL/ws/avancore_ws.1cws?wsdl";
+	АдресПродВИМ = "http://amapfront/PROD_AVC_FO-DL/ws/avancore_ws.1cws?wsdl";
+	АдресПродБФ = "http://AMAPFRONT/PROD_BFAM_FO-DL/ws/avancore_ws.1cws?wsdl";
 	
 	Попытка